explorer/lib/explorer_web/components/nav.ex

```diff
@@ -18,20 +18,28 @@ defmodule NavComponent do
     }>
       <div class="gap-x-6 inline-flex">
         <.link
-          class="text-3xl hover:scale-105 transform duration-150 active:scale-95"
+          class={
+            classes([
+              "hover:scale-105",
+              "transform",
+              "duration-150",
+              "active:scale-95",
+              active_view_class(assigns.socket.view, ExplorerWeb.Home.Index)
```

Comment:
I would not change the logo size, it looks really weird.

Reply:
+1 , i would experiment other effects. i.e. making it brighter or underline it

Reply:
If brighter, I think we'd need better contrast, like a much darker default and much brighter highlight. I think the underline would be easier.

Reply:
I will try using the underline. I have tried other effects, but they didn't seem to have any impact, possibly because it is an icon, not text.